mesheryctl/internal/cli/root/model/error.go

```diff
@@ -15,6 +15,20 @@ const (
 	ErrModelInitCode                    = "mesheryctl-1148"
 	ErrModelUnsupportedVersionCode      = "mesheryctl-1149"
 	ErrModelBuildCode                   = "mesheryctl-1151"
+
+	// Error Constants
+	ErrBuildUsage = "Usage:\nmesheryctl model build [model-name]\nor\nmesheryctl model build [model-name]/[model-version]\n\nRun 'mesheryctl model build --help' to see detailed help message"
+	ErrBuildFolderNotFound = "\nfolder %s does not exist"
+	ErrBuildMultiVersionNotSupported = "\nCommand does not support multiple versions build under one image"
+	
+	ErrInitOneArg = "must provide only one argument: model name"
+	ErrInitUnsupportedFormat = "[ %s ] are the only format supported"
+	ErrInitInvalidVersion = "version must follow a semver format, f.e. v1.2.3"
+	ErrInitFolderExists = "folder %s exists, please specify different model name or version"
+	ErrInitInvalidModelName = "invalid model name: name must match pattern ^[a-z0-9-]+$"
```

Comment:
I think the error message should be more explicit for non developers 
```suggestion
	ErrInitInvalidModelName = "invalid model name: name must contains alphanumeric characters or hyphens"
```